examples/motoko_examples/hello/canisters/motoko/main.mo

```diff
@@ -0,0 +1,7 @@
+actor {
+
+  public func greet(name : Text) : async Text {
```

Comment:
Should this be a query function?

Reply:
That's how it is the motoko examples https://github.com/dfinity/examples/blob/master/motoko/hello/src/hello/main.mo
Should we change all of them to be updates?